### DISCUSSION TITLE
Can it upload files from client?

### DISCUSSION THREAD

**Discussion by ghost** [framesurge/perseus#208] [General]
Can it upload files through browsers to server side?

**Comment by arctic-hen7** [comment]
Sure thing! There are all sorts of ways to do this in JS, and any of them should be accessible through `web-sys`, which Perseus uses behind the scenes. For example, you could make a `POST` request to a server with a file, or you could use a form, etc. What exactly is your use-case?

**Reply by ghost** [reply]
Can we use std::io and std::fs in Rust standard library?

**Reply by arctic-hen7** [reply]
No, since the network and filesystem are not accessible in the browser. You'll need to use `web_sys` for all this.

**Reply by ghost** [reply]
How to use web_sys to upload files?

**Reply by arctic-hen7** [reply]
Not sure off the top of my head, but anything you can do in JS, you can just about do in `web_sys`. I'm happy to create an example though in the next few days.
